### PR TITLE
[new downloader] Moving maps with error status to downloaded section of downloader.

### PR DESCRIPTION
**Pull request by bykoianko**
https://jira.mail.ru/browse/MAPSME-401

### CODE REVIEW

**Comment by bykoianko** [comment]
@syershov PTAL

File: storage/storage.cpp [hunk 1]
```diff
@@ -1137,43 +1131,17 @@ void Storage::GetChildrenInGroups(TCountryId const & parent,
...
-    child.ForEachDescendant([&](TCountryTreeNode const & descendant)
-                            {
-                              TCountryId const & countryId = descendant.Value().Name();
-                              if (!hasDownloadedDescendant && HasCountryId(localMaps, countryId))
-                                hasDownloadedDescendant = true;
-                            });
-    if (hasDownloadedDescendant == true)
-      downloadedChildren.push_back(childCountryId);
+    NodeStatus const childStatus = GetNodeStatus(childNode).status;
+    TCountryId const childValue = childNode.Value().Name();
```
Comment (trashkalmar) [inline storage/storage.cpp]: TCountryId consy & ?

**Comment by ygorshenin** [comment]
LGTM

**Comment by syershov** [comment]
LGTM

File: storage/storage.cpp (hunk 1 above)
Comment (bykoianko) [inline storage/storage.cpp]: Ага.

**Comment by bykoianko** [comment]
@trashkalmar PTAL&merge

**Pull request merged**